python/cugraph-pyg/cugraph_pyg/sampler/cugraph_sampler.py

```diff
@@ -11,19 +11,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# cuGraph is required
 try:
-    from cugraph_service.client.remote_graph_utils import import_optional, MissingModule
+    import cugraph
 except ModuleNotFoundError:
-    try:
-        from cugraph.utilities.utils import import_optional, MissingModule
-    except ModuleNotFoundError:
-        raise ModuleNotFoundError(
-            "cuGraph extensions for PyG require cuGraph"
-            "or cuGraph-Service to be installed."
-        )
-
-from cugraph_pyg.loader.dispatch import call_cugraph_algorithm
+    raise ModuleNotFoundError(
+        "cuGraph extensions for PyG require cuGraph to be installed."
+    )
```

Comment:
same question as the above import change

Reply:
fixed